test/mocks/mockCourseAnalysis.js

```diff
@@ -1,4 +1,4 @@
-const mockCourseAnalysis = (semester = '20172', roundIdList = '1') => ({
+const mockCourseAnalysis = (semester = '20172', roundIdList = '1', applicationCodes = '1') => ({
```

Comment:
`roundIdList` has to be removed, in order to be sure tests work

Reply:
roundIdList was removed from everywhere